TombEngine/Scripting/Internal/TEN/Objects/ObjectsHandler.h

```diff
@@ -64,6 +64,45 @@ class ObjectsHandler : public ScriptInterfaceObjectsHandler
 			return std::make_unique<R>(std::get<R::IdentifierType>(m_nameMap.at(name)));
 	}
 
+	template <typename R, char const* S>
+	std::vector <std::unique_ptr<R>> GetMoveablesBySlot(GAME_OBJECT_ID objID)
+	{
+		std::vector<std::unique_ptr<R>> items = {};
+		for (auto& [key, val] : m_nameMap)
+		{
+			if (!std::holds_alternative<std::reference_wrapper<MESH_INFO>>(val))
+			{
+				auto* item = &g_Level.Items[GetIndexByName(key)];
```

Comment:
I would use references instead of pointers where possible since there's an ongoing effort to make such conversions. `const auto& item = g_Level.Items[GetIndexByName(key)];` works just fine. :)

Reply:
done

Reply:
@Sezzary 
I cant do this, because it does NOT work with reference.. only pointer will work.
If I try this, I got compile error:

Fehler    C2440    "Initialisierung": "_Ty " kann nicht in "_Ty&" konvertiert werden